salt/metalk8s/orchestrate/downgrade/precheck.sls

```diff
@@ -16,6 +18,16 @@ Correct saltenv "{{ saltenv }}" for downgrade to "{{ dest_version }}":
 
 {%- endif %}
 
+{%- set dest_minor_version, _ = (dest_version|string).rsplit('.', 1) %}
+{%- set current_minor_version, _ = (expected|string).rsplit('.', 1) %}
+
+{%- if not metalk8s.downgrade.enabled and dest_minor_version != current_minor_version %}
+
+Downgrade from {{ current_minor_version }} to {{ dest_minor_version }} is not supported, because of etcd version change (see https://github.com/scality/metalk8s/releases/tag/2.6.0 for details):
+  test.fail_without_changes
```

Comment:
Maybe use a simpler state ID and explain why in a comment:

```suggestion
{%- set relnotes = 'https://github.com/scality/metalk8s/releases/tag/'
                 ~ current_minor_version ~ '.O' %}
Cannot downgrade from {{ current_minor_version }} to {{ dest_minor_version }}:
  test.fail_without_changes:
    - comment: |-
        Downgrade is not supported, because of etcd version change
        (see {{ relnotes }} for details)
```

(note that we can likely always link to release notes of the first release of the current minor, this will prevent us from forgetting about it in future versions - the reason why this downgrade path is not supported may evolve in the future, but we'll remember if it does... :crossed_fingers:)

Reply:
done.